### DISCUSSION TITLE
`with` statement to shorten variable lookups.

### DISCUSSION THREAD

**Discussion by jvanlangen** [dotnet/csharplang#6647] [Language Ideas] [answered]
I think it would be great to have something like the `with` statement we had in the early Pascal days:

For example:

    person.Info.Id = 10;
    person.Info.Name = "John";
    person.Info.Email = "a@b.c";

becomes:

    with (person.Info)
    {
        Id = 10;
        Name = "John";
        Email = "a@b.c";
    }

**Comment by HaloFour** [comment, marked as answer]
See: #1534
